notify-server/src/main/java/com/lexsus/ariaddna/client/ClientMessageProcessor.java

```diff
@@ -0,0 +1,16 @@
+package com.lexsus.ariaddna.client;
+
+import com.lexsus.ariaddna.server.MessageProcessor;
```

Comment:
Please change packet name to "com.stnetix.ariaddna.pushnotification.server.MessageProcessor"